setup.py

```diff
@@ -31,6 +31,7 @@ def find_version():
           'textparser>=0.21.1',
           'diskcache',
           'argparse_addons',
+          'typing_extensions>=3.10.0.0',
```

Comment:
possibly, this could be taken from `requirements.txt`? something like

```python
install_requires=open('requirements.txt').readlines(),
```

Reply:
(this might install a few extra dependencies, but IMO the reduced potential to shoot oneself into the foot more than makes up for this...)

Reply:
Please don't introduce unneeded dependencies in the `setup.py`. We care about install size.

An alternative, we could have a `requirments.txt` and a `requirments_dev.txt`? The requirments could then be included in `setup.py`

Reply:
Runtime dependencies should be defined in `setup.py` **only** (or pyproject.toml).